src/main/java/monitoringsystemturbo/model/timeline/Timeline.java

```diff
@@ -1,30 +1,32 @@
 package monitoringsystemturbo.model.timeline;
 
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
 import monitoringsystemturbo.model.app.ApplicationState;
 import monitoringsystemturbo.model.computer.ComputerStatistics;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Date;
-import java.util.List;
+import java.util.*;
 
 public class Timeline {
 
     protected Date datetimeStart;
     protected Date datetimeEnd;
     private ApplicationState lastApplicationState = ApplicationState.NOT_RUNNING;
-    protected ArrayList<Period> periods = new ArrayList<>();
+    protected ObservableList<Period> observableList;
```

Comment:
same thing with naming. Readability is better when name indicates what it's stored in a list